packages/icons/src/index.js

```diff
@@ -1,4 +1,19 @@
 export { default as __experimentalIcon } from './icon';
 
 export { default as check } from './library/check';
+export { default as column } from './library/column';
+export { default as columns } from './library/columns';
+export { default as comment } from './library/comment';
+export { default as cover } from './library/cover';
+export { default as file } from './library/file';
+export { default as gallery } from './library/gallery';
+export { default as group } from './library/group';
+export { default as html } from './library/html';
+export { default as image } from './library/image';
+export { default as list } from './library/list';
+export { default as mediaAndText } from './library/media-and-text';
```

Comment:
I wish we could use dashes instead of camelcase for this. I.e. `media-and-text` instaed of `mediaAndText`. But that's not possible, is it?

Reply:
not possible no, these are JS variables.

Reply:
😩
I know.